src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -633,6 +637,7 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
       throw new Error('performWork was called recursively.');
     }
     isPerformingWork = true;
+    isPerformingDeferredWork = priorityLevel >= HighPriority;
```

Comment:
Are we referring to animation as "deferred" work now too?
This makes sense to me but it's not very consistent with other places (e.g. `scheduleDeferredCallback`).